scipy/sparse/base.py

```diff
@@ -537,8 +558,33 @@ def __getattr__(self, attr):
         else:
             raise AttributeError(attr + " not found")
 
-    def transpose(self):
-        return self.tocsr().transpose()
+    def transpose(self, axes=None, copy=False):
+        """
+        Reverses the dimensions of the sparse matrix.
+
+        Parameters
+        ----------
+        axes : None, optional
+            This argument is in the signature *solely* for NumPy
+            compatibility reasons. Do not pass in anything except
+            for the default value, as reversing the dimensions is
```

Comment:
The phrase starting at the comma on this line is awkward, and can probably be removed entirely.